go/datas/pull_test.go

```diff
@@ -272,7 +267,7 @@ func (suite *PullSuite) TestPullDivergentHistory() {
 //                         \ -3-> L2 -1-> N
 //                                 \ -2-> L1 -1-> N
 //                                         \ -1-> L0
-func (suite *PullSuite) TestPullUpdates() {
+func (suite *PullSuite) SkipTestPullUpdates() {
```

Comment:
Rather than keep this optimization and fix this test, I'd rather just push forward towards the new sync world of not using the "chunk height diff" heuristic.